edgePy/_DGEList.py

```diff
@@ -116,10 +116,20 @@ def samples(self) -> np.array:
 
     @samples.setter
     def samples(self, samples: Optional[np.ndarray]) -> None:
-        # TODO: Validate samples here
-        # - Samples same length as ncol(self.counts) if defined
+        """Validate setting ``DGEList.samples`` for the illegal conditions:
+
+            * Must be the same length as the columns in counts`
+
+        Args:
+            samples: 1D string array representing identifiers of count columns
+
+        """
         if samples is not None:
```

Comment:
Can delete this line and dedent 128-129 since the expression also exists in the nested expression currently on 128.